ckanext/dcatapchharvest/harvesters_i14y.py

```diff
@@ -0,0 +1,206 @@
+import json
+
+import ckan.plugins as p
+import ckan.model as model
+from ckan.logic import NotFound, get_action
+
+from ckanext.dcat.harvesters.rdf import DCATRDFHarvester
+from ckanext.dcat.interfaces import IDCATRDFHarvester
+from ckanext.dcatapchharvest.dcat_helpers import get_pagination
+from ckanext.dcatapchharvest.harvest_helper import (
+    map_resources_to_ids,
+    check_package_change,
+    create_activity,
+)
+
+import logging
+log = logging.getLogger(__name__)
+
+
+class SwissDCATI14YRDFHarvester(DCATRDFHarvester):
```

Comment:
Can we add this class to the existing file `harvesters.py` instead of a new file?

Reply:
done